### ISSUE TITLE
[GC] I can't load any game with any Core

### ISSUE THREAD

**Issue by Ploggy** [libretro/RetroArch#13264]
I can't seem to load any Game with any Core with GC Retroarch?
I can boot the Core fine, but when I try to then boot a Game the Gamecube restarts?

**Comment by Papy-G** [comment]
Didn't try between 1.8.5 and 1.9.2 as my SD card died, but it is totally broken now, nothing goes over the core selection without rebooting the console.

I have three GC hardware variants (DOL-001, DOL-101, and RVL-001), and various video outputs options (RF/CVBS/RGB/YUV/HDMI).

One big problem for too long is the default video line count that gets off screen.
It would be nice to simplify the video modes options to only TV standard ones, or even pass-through modes, for TV consoles cores.

**Comment by Littlemac123** [comment]
my dude this is a dupe of this post i posted first DAYS ago and no one replied to it

**Comment by Papy-G** [comment]
No, this isn't, your issue is specific to nestopia, other cores know what files they can open.

**Comment by Littlemac123** [comment]
> No, this isn't, your issue is specific to nestopia, other cores know what files they can open.

and No this isn't. if you even saw that # post then you would know the screenshot i used was quick nes and that i said even .gb with things like gambatte. I dont think you read anything said there at all. I never ones even mentioned nestopia

**Comment by Papy-G** [comment]
I always read carefully whatever I refer to.

Maybe it is not specific to nestopia, but there are two distinct issues:

1. Cores that don't manage to be linked to their games, so you are stuck to the core selection menu with a blank list.

2. Cores that can recognise their games, but reboot the console upon core selection.

**Comment by twinaphex** [comment]
Does the Wii version still work?

**Comment by Littlemac123** [comment]
> Does the Wii version still work?

yes it does flawlessly :)

**Comment by Papy-G** [comment]
Unless: "NGC/Wii: Several improvements", as of 1.10.1, it still can't find the core to load the game with.
Running the core seems possible without selecting a rom file, tested with theodore.

Is there something we can do on the user's side? Building a linking file somewhere maybe?

**Comment by Littlemac123** [comment]
> Unless: "NGC/Wii: Several improvements", as of 1.10.1, it still can't find the core to load the game with. Running the core seems possible without selecting a rom file, tested with theodore.
> 
> Is there something we can do on the user's side? Building a linking file somewhere maybe?

im telling you thats lit what this is lol https://github.com/libretro/RetroArch/issues/13209

**Comment by Littlemac123** [comment]
its funny how this says https://www.libretro.com/index.php/retroarch-1-10-1-release/
**Good news for Gamecube/Wii users – after some downtime, RetroArch should work again on these platforms!**

I mean they did say "Should" haha

**Comment by Ploggy** [comment]
Tried updating to the latest nightly,, and the issue is still present.

**Comment by Ploggy** [comment]
After some trial and error it turns out if you disable Always Allow Core on Run Content in Settings/Core I can get some Cores to load games.. BUT only if I load the same Core/Content twice 😛 First time loading Core/Content will restart the GC like before, but after the Console restarts if I again load the same Core then load the same Content it will continue to load the game correctly? I'm not sure why it does this.. something to do with the Content not being recognised by the Core. I know normally the core is put into a txt file called Salamander.cfg? perhaps its not placed there until after Content load, but needs to be added after the Core dol is loaded?? sorry I'm just guessing here I don't know.

But still pretty good news huh? the GC port works.. kinda 😛

There is another issue too, once Content has loaded it feels like every Button has autofire mapped or something? if I hold ANY button it will just keep pressing the same bind making playing the game next to impossible.

Even with the load twice method some Cores still refuse to load Content or will DSI. It could be due to Memory issues like some people have said before, again I don't know.

I found out its possible to compress GC dol files using a tool called dolxz by fix94 similar to what was done on the WiiU, it has reduced the overall dol size quite abit..
Example with Snes9x.. Before = 4.75 MB After = 1.24 MB pretty good considering the GC needs all the free space it can get 🙂
I have used this tool with all the Dol files I've been testing, I don't know if this is a another reason why its started to work also.

**Comment by Papy-G** [comment]
Not sure compressed DOLs are kept compressed in the GCN's RAM after loading.

Tested Tomson's Theodore emulator from the 1.15.0 build with the excellent Mission:Liftoff, not sure How autofire may affect it, but nothing unexpected or annoying to report.

Doesn't seem to be older settings file, as Video output reverted back to its weird default (stretched to over screen at the bottom).
